### DISCUSSION TITLE
👋 Welcome to the AI4LAM teaching and learning discussion forum 👋

### DISCUSSION THREAD

**Discussion by davanstrien** [AI4LAM/TeachingAndLearning#5] [General]
Hello everyone, and welcome to our discussion forum focused on sharing resources and asking questions related to the use of machine learning in the Galleries, Libraries, Archives, and Museum (GLAM) sector! We hope that a forum like this can help the GLAM community learn from each other about how to use machine learning in the sector (and when not to use it!). 

The goal of this forum is to provide a space for people to come together and share their knowledge and expertise on machine learning in the GLAM sector. Whether you are just starting out or consider yourself an expert, we believe that everyone has something to contribute, and we encourage you to get involved in the discussion.  We don't want to try and replace existing resources and communities focused on machine learning but we think there is a valuable role for a space to discuss some of the domain-specific challenges of using machine learning in the GLAM sector. 

To make a post on this forum, you will need a GitHub account (you can create one [here](https://github.com/signup)). You can also find [docs](https://docs.github.com/en/discussions) on GitHub discussion forums. This will provide you with all the information you need to get started, including how to create a new topic, how to reply to existing topics, and how to use the various features of the platform.

We want to invite everyone to introduce themselves to the forum! Please tell us who you are, where you work, what your interest in machine learning and GLAM is, and if you have any particular resources or projects you'd like to share. We believe that by getting to know each other better, we can build a stronger and more supportive community that will benefit us all.

Once again, welcome to our discussion forum! We look forward to hearing from you all and hope this will be a valuable resource for everyone interested in machine learning in the GLAM sector.

**Comment by davanstrien** [comment]
Replying to my own intro post 😅

Hi, I'm Daniel van Strien 👋🏻 I work as a Machine Learning Librarian at [Hugging Face](https://huggingface.co/). I'm very interested in how machine learning can support the work of GLAM organisations and how machine learning can be responsibly deployed. I believe that GLAM organisations should favour open vs closed approaches to machine learning. I also think there is much more scope for organisations to collaborate to create machine learning datasets and models that benefit a broader range of users and use cases. 

I have been involved in the AI4LAM community (especially the teaching and learning working group) for a little while and have also been involved in some other educational initiatives related to making machine learning more accessible for people working in/with GLAM collections. These include an in-progress library carpentry lesson [Intro to AI for GLAM](https://carpentries-incubator.github.io/machine-learning-librarians-archivists/), a Programming Historian lesson [Computer Vision for the Humanities: An Introduction to Deep Learning for Image Classification](https://programminghistorian.org/en/lessons/computer-vision-deep-learning-pt1), and a Jupyter Book ['Classifying 19th Century British Library books using Crowdsourcing and Machine Learning'](https://living-with-machines.github.io/genre-classification/intro.html). 

I'm also involved in [BigLAM](https://huggingface.co/biglam), an initiative focused on making GLAM datasets related to machine learning more accessible. 

I'm excited to meet other people working at the intersection of machine learning and GLAM 🤗

**Comment by mialondon** [comment]
Hi everyone! And thanks Daniel for getting us going!

I'm Mia, I'm currently Digital Curator for Western Heritage [Collections at the British Library](https://bl.uk/digital) and a Co-Investigator on the [Living with Machines](https://livingwithmachines.ac.uk/) project.

Like Daniel (and many others) I'm interested in the responsible deployment of AI (machine learning) in GLAMs. I think our attention and money is a vote for certain values, so we should choose our tools wisely. I've been involved in the 'musetech' or digital heritage field for some years, and have been wondering what AI tools will mean for public engagement with collections via crowdsourcing for almost ten years now.

I'm curious to hear what others are up to, the questions you have, what excites or scares you!
